core/domain/question_domain.py

```diff
@@ -296,3 +331,41 @@ def to_dict(self):
             'question_id': self.question_id,
             'skill_id': self.skill_id,
         }
+
+
+class QuestionRights(object):
+    """Domain object for question rights."""
+
+    def __init__(self, question_id, creator_id):
+        """Constructs a QuestionRights domain object.
+
+        Args:
+            question_id: str. The id of the question.
+            creator_id: str. The id of the user who has initially created
+                the question.
+        """
+        self.id = question_id
+        self.creator_id = creator_id
+
+    def to_dict(self):
+        """Returns a dict suitable for use by the frontend.
+
+        Returns:
+            dict. A dict representation of QuestionRights suitable for use
+                by the frontend.
+        """
+        return {
+            'question_id': self.id,
+            'creator_id': self.creator_id
+        }
+
+    def is_creator(self, user_id):
+        """Checks whether given user is a manager of the question.
```

Comment:
Just to check -- manager = creator? (See line 369)

Reply:
Yeah, renamed to creator.